openedx/core/djangoapps/schedules/management/commands/sync_schedules_to_course_start.py

```diff
@@ -0,0 +1,54 @@
+"""
+Support "fixing" schedules that currently have start dates prior to the course start date.
+
+This can happen if a course start date is moved into the future and the corresponding
+schedules are not updated for any reason.
+"""
+from __future__ import print_function
+
+import logging
+import textwrap
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+
+LOG = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Support "fixing" schedules that currently have start dates prior to the course start date.
+
+    This can happen if a course start date is moved into the future and the corresponding
+    schedules are not updated for any reason.
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument('--dry-run', default=False, action='store_true')
+        parser.add_argument('from_date')
+        parser.add_argument('course_id')
+
+    def handle(self, *args, **options):
+        query = textwrap.dedent(
+            """
```

Comment:
Did you test dry-run mode? I think you might need the following patch to make it print correctly (or else we just have a lot of cargo-cult across the platform)
```suggestion
            """\
```

Reply:
I did test with dry-run, I think it's cargo-cult? I mean, one could argue it prints with an extra new-line at the beginning, but dry-run is functional.